scipy/sparse/linalg/_expm_multiply.py

```diff
@@ -203,15 +205,16 @@ def expm_multiply(A, B, start=None, stop=None, num=None,
     >>> expm(2*A).dot(B)                # Verify 3rd timestep
     array([ 2.71828183,  1.        ])
     """
+    rng = np.random.default_rng(rng)
```

Comment:
Personally, I think it is OK to just do this, since the randomness was undocumented and shouldn't matter.
As a reviewer, though, I thought I should mention that there may be users who discovered that `np.random.seed` could control the random state, and this will break that.